apps/docs/src/guide/quickstart/index.md

```diff
@@ -0,0 +1,87 @@
+# Quickstart
+
+You can quickly bootstrap a full-stack Fuel project locally with the following command:
+
+::: code-group
+
+```sh [pnpm]
+pnpm create fuels
```

Comment:
Please remove all mentions of `bun` and `yarn` from the Quickstart:

 - We removed the `yarn` in https://github.com/FuelLabs/fuels-ts/pull/1470.
 - We can reevaluate `bun` later